### ISSUE TITLE
[iOS][3.6.3] Error getting value from 'LogLevel' on 'DevToDev.Analytics.DTDAnalyticsConfiguration'

### ISSUE THREAD

**Issue by evgeny-detkov** [devtodev-analytics/Unity-sdk-3.0#10]
Hello!

Using Unity 2020.3.48f1, updated Devtodev from 3.4.0 to 3.6.3.
Everything is ok with the Android platform, there are no problems: the SDK is initialized, events are sent and appear in the dashboard.

But there is a problem with the iOS version: there is an error when trying to initialize (see screenshot). Screenshot taken from iOS-device in Touchconsole-plugin. Using Xcode 14.2 on Macmini M1.

Intermediate versions of Devtodev SDK did not check.
I can provide more if needed.

![image](https://github.com/devtodev-analytics/Unity-sdk-3.0/assets/5319984/1c93a292-d2da-4989-871b-391127e11246)

**Comment by kaikristo** [comment]
Hello, which method do you use for initialization? With ab tests? With config? Can you provide me your initialization code without appkey?

**Comment by evgeny-detkov** [comment]
Oh, I'm sorry, I solved the problem the other day. I downgraded the Devtodev SDK to version 3.5.0, and in this version there was a more extended log, there was an error in the stacktrace with Newtonsoft.Json. The current default version after the engine update on 2020.3.48 was 3.2.0. I checked the Package Manager, and there is an update to 3.2.1. It saved. Now everything is working stably, initialization is taking place, events are being sent.
Thanks for understanding!
